Lib/dataclasses.py

```diff
@@ -1158,6 +1160,14 @@ class C(Base):
 
     For the bases and namespace parameters, see the builtin type() function.
```

Comment:
A little improve :-)

```suggestion
    For the bases and namespace parameters, see the built-in type() function.
```

Reply:
I did not change this line and removing a space would misalign it. `built-in` and `builtin` seems both used in the rest of the code.